test/test_cpp20/clang_only.cpp

```diff
@@ -0,0 +1,28 @@
+#include "pch.h"
+#include <winrt/Windows.Storage.Pickers.h>
+
+#ifdef __clang__
+
+using namespace winrt;
+using namespace Windows::Foundation;
+using namespace Windows::Storage::Pickers;
+
+// winrt::fire_and_forget PickImage()
+// {
+//     FileOpenPicker picker{};
+//     picker.ViewMode(PickerViewMode::Thumbnail);
+//     picker.FileTypeFilter().Append(L".png"); // <--- crash occurred here
+// }
+
+TEST_CASE("clang_library")
+{
+    // PickImage();
+
+    FileOpenPicker picker{};
+    picker.ViewMode(PickerViewMode::Thumbnail);
+    picker.FileTypeFilter().Append(L".png"); // <--- crash occurred here
+
+    REQUIRE(true);
+}
```

Comment:
Maybe add a comment what this is doing/testing & why there is code commented out

Reply:
[sigh] I totally did last night but my edits seem to have been lost.  Re-adding them.